eng/common/scripts/Verify-Links.ps1

```diff
@@ -30,6 +30,10 @@ $emptyLinkMessage = "There is at least one empty link in the page. Please replac
 if (!$userAgent) {
   $userAgent = "Chrome/87.0.4280.88"
 }
+if (!$branchReplaceRegex) {
```

Comment:
Instead of calling git from this script I recommend we just remove the default value here as it isn't needed for basic usage anyway. We can simply add that information to the documentation for this parameter. We should actually also take this opportunity to use the PS help comment syntax to document this script. See https://docs.microsoft.com/en-us/powershell/scripting/developer/help/examples-of-comment-based-help?view=powershell-7.1#example-2-comment-based-help-for-a-script.

Reply:
I actually need to start doing that for more of my scripts as well :)